subprojects/logging/src/main/java/org/gradle/internal/logging/events/UpdateNowEvent.java

```diff
@@ -34,6 +34,11 @@ public long getTimestamp() {
     }
 
     @Override
+    public String toString() {
+        return UpdateNowEvent.class.getSimpleName() + " " + timestamp;
```

Comment:
What is this `toString()` method used for? Debugging only? Because calling `getSimpleName()` is very expensive.